e2e-tests/production-runtime/cypress/plugins/block-resources.js

```diff
@@ -0,0 +1,120 @@
+const fs = require(`fs-extra`)
+const path = require(`path`)
+const glob = require(`glob`)
+
+const publicDir = path.join(__dirname, `..`, `..`, `public`)
+
+const getAssetManifest = () => {
+  const { assetsByChunkName } = require(`${publicDir}/webpack.stats.json`)
+  return assetsByChunkName
+}
+
+const moveAsset = (from, to) => {
+  const fromExists = fs.existsSync(from)
+  const toExists = fs.existsSync(to)
+
+  if (fromExists && !toExists) {
+    fs.moveSync(from, to, {
+      overwrite: true,
+    })
+  }
+}
+
+const getAssetPath = assetFileName => path.join(publicDir, assetFileName)
+const getHiddenAssetPath = assetFileName => getAssetPath(`_${assetFileName}`)
+
+const restoreAsset = assetFileName => {
+  moveAsset(getHiddenAssetPath(assetFileName), getAssetPath(assetFileName))
+}
+
+const blockAsset = assetFileName => {
+  moveAsset(getAssetPath(assetFileName), getHiddenAssetPath(assetFileName))
+}
+
+const blockAssetsForChunk = ({ chunk, filter }) => {
+  const assetManifest = getAssetManifest()
+  assetManifest[chunk].forEach(blockAsset)
+  console.log(`Blocked assets for chunk "${chunk}"`)
```

Comment:
I'm unsure what the difference is but I would go for cypress.log 🤷‍♂  (same for the others below)
https://docs.cypress.io/api/commands/log.html#Syntax
```suggestion
  cy.log(`Blocked assets for chunk "${chunk}"`)
```

feel free to ignore

Reply:
Nice! Didn't know about that one

Reply:
Ahh, except that `plugins/block-resources.js` runs in node, not the browser. So no, we can't use this.

Reply:
Thanks for checking!